### PR TITLE
Add a convinvscale op, related instances and examples

### PR DESCRIPTION
**Pull request by geyyer**
- Update the element op
- Add an example and a client example
- Add instances

### CODE REVIEW

File: example/62_convnd_activ/convinvscale/convnd_fwd_convinvscale_common.hpp [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    copy(wei_g_k_c_xs_desc.GetStrides(), b_g_k_c_xs_strides);
+    copy(out_g_n_k_wos_desc.GetLengths(), e_g_n_k_wos_lengths);
+    copy(out_g_n_k_wos_desc.GetStrides(), e_g_n_k_wos_strides);
+    copy(conv_param.conv_filter_strides_, conv_filter_strides);
+    copy(conv_param.conv_filter_dilations_, conv_filter_dilations);
+    copy(conv_param.input_left_pads_, input_left_pads);
+    copy(conv_param.input_right_pads_, input_right_pads);
+
+    // sample scale values, same as host
+    float scale_in  = 3.3f;
```
Comment (zjing14) [inline example/62_convnd_activ/convinvscale/convnd_fwd_convinvscale_common.hpp]: Based on the design of this example, we should change scales per run.

File: library/include/ck/library/tensor_operation_instance/gpu/grouped_conv_fwd/device_grouped_conv_fwd_xdl_convinvscale_instance.hpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+#ifdef CK_ENABLE_FP8
+        // generic instance
+        DeviceGroupedConvFwdMultipleABD_Xdl_CShuffle<NDimSpatial,ALayout,BLayout,  DsLayout,ELayout,    F8,    F8,     F32,      F32,  Tuple<>,    F8, PassThrough, PassThrough,     ConvInvscale,       ConvSpec, GemmMNKPadding,        1,    64,    64,    64,    32,   8,   8,   32,   32,    2,    2,     S<4, 16, 1>,     S<1, 0, 2>,     S<1, 0, 2>,              2,              1,              8,         1,     S<4, 16, 1>,     S<1, 0, 2>,     S<1, 0, 2>,             2,              1,              8,         1,           1,           1,               S<1, 16, 1, 4>,               1,     F8,      F8>,
```
Comment (zjing14) [inline library/include/ck/library/tensor_operation_instance/gpu/grouped_conv_fwd/device_grouped_conv_fwd_xdl_convinvscale_instance.hpp]: We can reuse instances between scale and invscale by making OutElementOp as a template argument.

File: example/62_convnd_activ/convinvscale/convnd_fwd_convinvscale_common.hpp (hunk 1 above)
Comment (geyyer) [reply example/62_convnd_activ/convinvscale/convnd_fwd_convinvscale_common.hpp]: Added in [4775de0](https://github.com/ROCm/composable_kernel/pull/1307/commits/4775de0ed006d3d9cf615834b380bba5d91d4ddd)

File: library/include/ck/library/tensor_operation_instance/gpu/grouped_conv_fwd/device_grouped_conv_fwd_xdl_convinvscale_instance.hpp (hunk 2 above)
Comment (geyyer) [reply library/include/ck/library/tensor_operation_instance/gpu/grouped_conv_fwd/device_grouped_conv_fwd_xdl_convinvscale_instance.hpp]: Added in [be42d56](https://github.com/ROCm/composable_kernel/pull/1307/commits/be42d56acd7704cefbd64cd6984ec42e6bdc3b16)

File: client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+{
+    std::size_t in_mem_size  = GetInputByte<InDataType, NumDimSpatial>(in_lengths);
+    std::size_t wei_mem_size = GetWeightByte<WeiDataType, NumDimSpatial>(wei_lengths);
+    std::size_t out_mem_size = GetOutputByte<OutDataType, NumDimSpatial>(out_lengths);
+
+    SimpleDeviceMem in(in_mem_size);
+    SimpleDeviceMem wei(wei_mem_size);
+    SimpleDeviceMem out(out_mem_size);
+
+    float scale_in;
```
Comment (zjing14) [inline client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp]: It seems that scales are not set values.

Comment (geyyer) [reply client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp]: Thanks for noticing, let me fix it!

Comment (geyyer) [reply client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp]: Added in [a39c6d7](https://github.com/ROCm/composable_kernel/pull/1307/commits/a39c6d737ad8bd63696d39c04e7c589bc6e5500e)

File: client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+    std::size_t in_mem_size  = GetInputByte<InDataType, NumDimSpatial>(in_lengths);
+    std::size_t wei_mem_size = GetWeightByte<WeiDataType, NumDimSpatial>(wei_lengths);
+    std::size_t out_mem_size = GetOutputByte<OutDataType, NumDimSpatial>(out_lengths);
+
+    SimpleDeviceMem in(in_mem_size);
+    SimpleDeviceMem wei(wei_mem_size);
+    SimpleDeviceMem out(out_mem_size);
+
+    float scale_in = float(std::rand()) / float(RAND_MAX);
+    ;
```
Comment (zjing14) [inline client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp]: Redundant, please remove it.

Comment (geyyer) [reply client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp]: Wow, this is some wild formatting!

Comment (geyyer) [reply client_example/24_grouped_conv_activation/grouped_convnd_fwd_convinvscale/common.hpp]: Fixed in [e97c444](https://github.com/ROCm/composable_kernel/pull/1307/commits/e97c444e2af4e587023029836269ed5c383c5b73)

**Review by zjing14 — APPROVED** [review]

**Pull request merged**